<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>329</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>50</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollAreaForRoutes">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>69</width>
         <height>246</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollAreaForStations">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>69</width>
         <height>246</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>474</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Изменить</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Изменить информацию о маршрутах</string>
     </property>
     <widget class="QMenu" name="menu_3">
      <property name="title">
       <string>Изменить маршрут</string>
      </property>
      <addaction name="action_5"/>
      <addaction name="action_6"/>
      <addaction name="action_7"/>
     </widget>
     <addaction name="action_2"/>
     <addaction name="menu_3"/>
     <addaction name="action_4"/>
    </widget>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Изменить информацию о станциях</string>
     </property>
     <addaction name="action_9"/>
     <addaction name="action_10"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="menu_4"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_2">
   <property name="text">
    <string>Добавить маршрут</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Удалить маршрут</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Добавить станцию</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Изменить название станции</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Удалить станцию</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Добавить информацию о станции</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Удалить информацию о станции</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
